cmd/root.go

```diff
@@ -239,6 +249,28 @@ func getSurveyorOpts() *surveyor.Options {
 	opts.JetStreamConfigDir = viper.GetString("jetstream")
 	opts.Accounts = viper.GetBool("accounts")
 
+	switch logLevel := strings.ToLower(viper.GetString("log-level")); logLevel {
```

Comment:
Should probably call out in the readme that the normal logrus env var doesn't work here. You need NATS_SURVEYOR_LOG-LEVEL as the env var instead of just LOG_LEVEL. Just in case someone sees logrus and expects the normal env var to work.

Reply:
nevermind, there isn't a default env var for logrus. Just remembering incorrectly.

Reply:
put example in README